integrations/kml-parser/src/main/java/ch/stayhome/integrations/kml/job/StayHomeEntry.java

```diff
@@ -0,0 +1,49 @@
+package ch.stayhome.integrations.kml.job;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class StayHomeEntry {
+
+	@NotBlank
+	private String id;
+
+	@NotBlank
+	private String providerName;
+
+	@NotBlank
+	private String name;
+
+	@NotBlank
+	private String description;
+
+	@NotBlank
+	private String contact;
+
+	@NotBlank
+	private String address;
+
+	@NotBlank
+	private String location;
+
+	private String email;
```

Comment:
When you have fields that can be `null`, I'd suggest to use getters returning Optional:

```java
private String email;

public Optional<String> getEmail() {
    return Optional.ofNullable(email);
}
```